packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -126,54 +129,37 @@ function PaymentMethodRadioItem({
 function FiatExchangeOptions({ route, navigation }: Props) {
   const { t } = useTranslation(Namespaces.fiatExchangeFlow)
   const isCashIn = route.params?.isCashIn ?? true
-  const {
-    KOTANI_SUPPORTED,
-    PONTO_SUPPORTED,
-    BITFY_SUPPORTED,
-    FLOW_BTC_SUPPORTED,
-  } = useCountryFeatures()
-  const pontoEnabled = useSelector(pontoEnabledSelector)
-  const kotaniEnabled = useSelector(kotaniEnabledSelector)
-  const bitfyUrl = useSelector(bitfyUrlSelector)
-  const flowBtcUrl = useSelector(flowBtcUrlSelector)
-  const showPonto = pontoEnabled && PONTO_SUPPORTED
-  const showKotani = kotaniEnabled && KOTANI_SUPPORTED
-  const showBitfy = bitfyUrl && BITFY_SUPPORTED
-  const showFlowBtc = flowBtcUrl && FLOW_BTC_SUPPORTED
-
-  Logger.debug(`Ponto: ${pontoEnabled} Kotani: ${kotaniEnabled}`)
+  const countryCode = useSelector(defaultCountryCodeSelector)
 
   const [selectedCurrency, setSelectedCurrency] = useState<CURRENCY_ENUM>(CURRENCY_ENUM.DOLLAR)
   const [selectedPaymentMethod, setSelectedPaymentMethod] = useState<PaymentMethod>(
-    isCashIn ? PaymentMethod.CARD : PaymentMethod.EXCHANGE
+    isCashIn ? PaymentMethod.Card : PaymentMethod.Exchange
   )
+  const [selectedLocalProvider, setSelectedLocalProvider] = useState<LocalCicoProvider>()
   const [isEducationDialogVisible, setEducationDialogVisible] = useState(false)
 
+  const asset = selectedCurrency === CURRENCY_ENUM.DOLLAR ? 'cusd' : 'celo'
+  const flow = isCashIn ? 'cashIn' : 'cashOut'
+
   const goToProvider = () => {
     ValoraAnalytics.track(FiatExchangeEvents.cico_option_chosen, {
       isCashIn,
       paymentMethod: selectedPaymentMethod,
       currency: selectedCurrency,
     })
-    if (selectedPaymentMethod === PaymentMethod.EXCHANGE) {
+    if (selectedPaymentMethod === PaymentMethod.Exchange) {
       navigate(Screens.ExternalExchanges, {
         currency: selectedCurrency,
       })
-    } else if (selectedPaymentMethod === PaymentMethod.PONTO) {
-      navigate(Screens.LocalProviderCashOut, { uri: PONTO_URI })
-    } else if (selectedPaymentMethod === PaymentMethod.KOTANI) {
-      navigate(Screens.LocalProviderCashOut, { uri: KOTANI_URI })
-    } else if (selectedPaymentMethod === PaymentMethod.BITFY && bitfyUrl) {
-      navigate(Screens.LocalProviderCashOut, { uri: bitfyUrl })
-    } else if (selectedPaymentMethod === PaymentMethod.FLOW_BTC && flowBtcUrl) {
-      navigate(Screens.LocalProviderCashOut, { uri: flowBtcUrl })
-    } else if (selectedPaymentMethod === PaymentMethod.GIFT_CARD) {
+    } else if (selectedPaymentMethod === PaymentMethod.LocalProvider && selectedLocalProvider) {
+      navigate(Screens.WebViewScreen, { uri: selectedLocalProvider[asset].url })
```

Comment:
I wonder for these local providers if it makes more sense to open them in an external browser. In the external browser users are more likely to already be logged in and even if they're not they are going to have to go back to Valora to copy their address which they won't be able to do if they're in a WebView in the app.

wdyt?

Reply:
That's a great point. I agree that an external browser would be preferable for all the reasons you stated. I think Nitya will agree but I'll check with her before making the change